src/main/java/mate/jdbc/dao/ManufacturersDaoImpl.java

```diff
@@ -0,0 +1,117 @@
+package mate.jdbc.dao;
+
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+import mate.jdbc.exception.DataProcessException;
+import mate.jdbc.lib.Dao;
+import mate.jdbc.model.Manufacturer;
+import mate.jdbc.util.ConnectionUtil;
+
+@Dao
+public class ManufacturersDaoImpl implements ManufacturersDao {
+    private static final int FIRST_PARAMETER = 1;
+    private static final int SECOND_PARAMETER = 2;
+    private static final int THIRD_PARAMETER = 3;
+
+    public Manufacturer create(Manufacturer manufacturer) {
+        String query = "INSERT INTO manufacturers(name,country) VALUES(?,?)";
+        try (Connection connection = ConnectionUtil.getConnection();
+                PreparedStatement statement = connection
+                        .prepareStatement(query, Statement.RETURN_GENERATED_KEYS)) {
+            statement.setString(FIRST_PARAMETER, manufacturer.getName());
+            statement.setString(SECOND_PARAMETER, manufacturer.getCountry());
+            statement.executeUpdate();
+            ResultSet generatedKeys = statement.getGeneratedKeys();
+            if (generatedKeys.next()) {
+                Long id = generatedKeys.getObject(FIRST_PARAMETER, Long.class);
+                manufacturer.setId(id);
+            }
+        } catch (SQLException e) {
+            throw new DataProcessException("Can't insert row to manufacturers with manufacturer id"
+                    + manufacturer.getId(), e);
+        }
+        return manufacturer;
+    }
+
+    @Override
+    public Optional<Manufacturer> get(Long id) {
+        String query = "SELECT id, name, country FROM manufacturers "
+                + "WHERE id = ? AND is_deleted = false";
+        try (Connection connection = ConnectionUtil.getConnection();
+                PreparedStatement statement = connection.prepareStatement(query)) {
+            statement.setLong(FIRST_PARAMETER, id);
+            ResultSet resultSet = statement.executeQuery();
+            return resultSet.next() ? Optional.of(parseManufacturer(resultSet)) : Optional.empty();
```

Comment:
But for readability is better :
```suggestion
            return resultSet.next() 
                ? Optional.of(parseManufacturer(resultSet)) 
                : Optional.empty();
```